STC-DCP 2nd phase/content/css/main.css

```diff
@@ -0,0 +1,292 @@
+@font-face {
+  font-family: "STC forward ";
+  src: url(/content/fonts/STC\ Forward\ Regular.woff);
```

Comment:
Please make sure you select the right **url** for the regular font ,
in this case it must be  `src: url(../fonts/STC\ Forward\ Regular.woff)`